NSS-Day-Cohort-42/rare-API-media-moguls
# Cb change active

#### Changes Made
1. modified views/user.py to add a change_active action
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout cb-change-active
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Run Django server
> go to Postman and use Get request on url `http://localhost:8000/users` and check current status of user's `is_active` property
> Use Patch request on url `http://localhost:8000/users/2/change_active`, you should get a 204 response
> Use Get request on url `http://localhost:8000/users` and check to see that user 2's `is_active` property has changed
> Note: Do NOT check with `http://localhost:8000/users/1/change_active` because it will delete the first user, which will invalidate the token authentication you have for Postman. If you do, then running `http://localhost:8000/users`will return a "detail": "User inactive or deleted." and you will need to run the nuke.sh command to fix.
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
